lib/routes/sms.js

```diff
@@ -58,8 +58,8 @@ module.exports = (log, db, config, customs, sms) => {
           .then(createSigninCode)
           .then(sendMessage)
           .then(logSuccess)
-          .then(createResponse)
-          .then(reply, reply)
+          .then(() => { return createResponse })
```

Comment:
@deeptibaghel everywhere you write `return createResponse` you actually have to call the function, such as

```
return createResponse()
```

Once you fix this everywhere (especially in `path: '/account/reset',` route) then password reset should work